.github/workflows/case-data-update.yml

```diff
@@ -0,0 +1,41 @@
+name: Update case data
+
+on:
+  push:
+    branches: [master]
+    paths: ["/data-serving/scripts/convert-data/**"]
+  # TODO: Remove this once we have a webhook that triggers the update.
+  schedule:
+    # Every day at 5AM (data is typically pushed a little after 4AM UTC).
+    - cron: "* 5 * * *"
+
+jobs:
+  update-case-data-prod:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Install MongoDB CLIs
+        run: |
+          wget -qO - https://www.mongodb.org/static/pgp/server-4.2.asc | sudo apt-key add -
+          echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.org/apt/ubuntu xenial/mongodb-org/4.2 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-4.2.list
+          sudo apt-get update
+          sudo apt-get install -y mongodb-org=4.2.6 mongodb-org-server=4.2.6 mongodb-org-shell=4.2.6 mongodb-org-mongos=4.2.6 mongodb-org-tools=4.2.6 libcurl3
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.x"
+
+      - name: Install Python dependencies
+        run: python -m pip install --upgrade pandas
+
+      - name: Run update script
+        run: ./data-serving/scripts/data-pipeline/convert_and_import_latest_data.sh -m mongodb+srv://${{ secrets.MONGODB_USER }}:${{ secrets.MONGODB_PASS }}@covid19-map-cluster01-sc7u9.mongodb.net
```

Comment:
Hmmm I'm not a fan of exposing the db url like that, why not having one secret db_connection_url instead that contains everything as we do in the other services?

Reply:
good call, thanks for pointing this out!